Unhandled Promise Rejection Warning

`[23.08.2021 23:07.42.319] [LOG]   Starting MagicMirror: v2.16.0
[23.08.2021 23:07.42.324] [LOG]   Loading config ...
[23.08.2021 23:07.42.331] [LOG]   Loading module helpers ...
[23.08.2021 23:07.42.333] [LOG]   No helper found for module: alert.
[23.08.2021 23:07.42.522] [LOG]   Initializing new module helper ...
[23.08.2021 23:07.42.523] [LOG]   Module helper loaded: updatenotification
[23.08.2021 23:07.42.524] [LOG]   No helper found for module: clock.
[23.08.2021 23:07.42.706] [LOG]   Initializing new module helper ...
[23.08.2021 23:07.42.707] [LOG]   Module helper loaded: calendar
[23.08.2021 23:07.42.708] [LOG]   No helper found for module: compliments.
[23.08.2021 23:07.42.708] [LOG]   No helper found for module: weather.
[23.08.2021 23:07.44.462] [LOG]   Initializing new module helper ...
[23.08.2021 23:07.44.463] [LOG]   Check MagicMirror version for node helper 'MMM-Ring' - Minimum version: 2.11.0 - Current version: 2.16.0
[23.08.2021 23:07.44.464] [LOG]   Version is ok!
[23.08.2021 23:07.44.465] [LOG]   Module helper loaded: MMM-Ring
[23.08.2021 23:07.44.465] [LOG]   All module helpers loaded.
[23.08.2021 23:07.44.556] [LOG]   Starting server on port 8080 ... 
[23.08.2021 23:07.44.568] [LOG]   Server started ...
[23.08.2021 23:07.44.570] [LOG]   Connecting socket for: updatenotification
[23.08.2021 23:07.44.571] [LOG]   Connecting socket for: calendar
[23.08.2021 23:07.44.572] [LOG]   Starting node helper for: calendar
[23.08.2021 23:07.44.573] [LOG]   Connecting socket for: MMM-Ring
[23.08.2021 23:07.44.621] [LOG]   MMM-Ring at (8/23/2021, 11:07:44 PM): Starting module: MMM-Ring
[23.08.2021 23:07.44.621] [LOG]   Sockets connected & modules started ...
[23.08.2021 23:07.44.897] [LOG]   Launching application.
[23.08.2021 23:07.47.363] [LOG]   Create new calendarfetcher for url: http://www.calendarlabs.com/ical-calendar/ics/76/US_Holidays.ics - Interval: 300000
[23.08.2021 23:07.47.411] [INFO]  Checking git for module: MMM-Ring
[23.08.2021 23:07.48.155] [INFO]  Calendar-Fetcher: Broadcasting 6 events.
[23.08.2021 23:07.48.361] [LOG]   MMM-Ring at (8/23/2021, 11:07:48 PM): Refresh Token Updated
[23.08.2021 23:07.48.902] [ERROR] (node:5285) UnhandledPromiseRejectionWarning: HTTPError: Response code 404 (Not Found)
    at Request.<anonymous> (/home/pi/MagicMirror/modules/MMM-Ring/node_modules/got/dist/source/as-promise/index.js:117:42)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[23.08.2021 23:07.48.903] [ERROR] (node:5285) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
[23.08.2021 23:07.48.905] [ERROR] (node:5285) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[23.08.2021 23:09.03.494] [LOG]   Shutting down server...
[23.08.2021 23:09.03.496] [LOG]   Stopping module helper: updatenotification
[23.08.2021 23:09.03.496] [LOG]   Stopping module helper: calendar
[23.08.2021 23:09.03.497] [LOG]   MMM-Ring at (8/23/2021, 11:09:03 PM): Stopping module helper: MMM-Ring
`

--- a/README.md
+++ b/README.md
@@ -56,7 +56,7 @@ To use this module, add the following configuration block to the modules array i
 
 (installed via `npm install` in the installation instructions above)
 
-- [ring-client-api](https://www.npmjs.com/package/ring-client-api) - version 8.2.0 or higher
+- [ring-client-api](https://www.npmjs.com/package/ring-client-api) - version 9.21.0 or higher
 - [hls.js](https://www.npmjs.com/package/hls.js/v/canary) - version 0.13.0 or higher
 - [rxjs](https://www.npmjs.com/package/rxjs) - version 6.5.5 or higher
 - [chokidar](https://www.npmjs.com/package/chokidar) - version 3.4.0 or higher

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "chokidar": "^3.5.1",
     "dotenv": "^8.6.0",
     "hls.js": "^0.13.0",
-    "ring-client-api": "^9.18.1",
+    "ring-client-api": "^9.21.0",
     "rxjs": "^6.6.7"
   },
   "repository": {

